src/frameworks/drupal7/drush.md

```diff
@@ -32,15 +62,19 @@ $ drush
 
 And see a list of available commands.
 
-> -   [Install Drush](https://github.com/drush-ops/drush)
+> -   [Drush repository on Github](https://github.com/drush-ops/drush)
 
 ## Use drush aliases
 
 ### Create Drush aliases
 
-[Drush aliases](http://drush.readthedocs.org/en/master/usage/index.html#site-aliases) make it easy to manage your development websites. Here's an example of a [Drush alias file](https://github.com/drush-ops/drush/blob/master/examples/example.aliases.drushrc.php).
+[Drush aliases](http://drush.readthedocs.org/en/master/usage/index.html#site-aliases) make it easy to manage your development websites. Here's an
+example of a [Drush alias
+file](https://github.com/drush-ops/drush/blob/master/examples/example.aliases.drushrc.php).
```

Comment:
Why the extra newlines here?

Reply:
Keeps it interesting. (I'll remove them)